fan-xiaomi.js

```diff
@@ -19,8 +19,29 @@ class FanXiaomi extends HTMLElement {
                 ui.querySelector('.var-title').textContent = this.config.name+' (Disconnected)';
                 return;
             }
+        }
+
+        const attrs = state.attributes;
+
+        let p5_speed_list = {"Level 1":1,"Level 2":35,"Level 3":70,"Level 4":100}
```

Comment:
```suggestion
        let p5_speed_list = [1, 35, 70, 100]
```
Perhaps using arrays, indexing them will be easier and more canonical later on in the speed selection, also mitigates typos